feature/policy_forwarding/encapsulation/otg_tests/static_encap_gue_ipv4/README.md

```diff
@@ -0,0 +1,334 @@
+Verify:
+
+*   Policy forwarding packet counters matches the packet count of traffic
+    generated from ATE:Port1.
+*   The packet count of traffic sent from ATE:Port1 should be equal to the sum
+    of all packets egressing DUT ports 2 to 5.
+*   All packets egressing DUT ports 2 to 5 are GUE encapsulated.
+*   ECMP hashing works (equal traffic) over the two LAG interfaces.
+*   LAG hashing works (equal traffic) over the two Singleton ports.
+*   ToS for all GUE encapsulated packets:
+    *   GUE header ToS is **0x80**.
+    *   Inner header ToS is **0x80**.
+*   TTL for all GUE encapsulated packets:
+    *   GUE header TTL is **9**.
+    *   Inner header TTL is **9**.
+
+### PF-1.3.2: IPv6 traffic GUE encapsulation without explicit ToS/TTL configuration on tunnel
+
+ATE action:
+
+*   Generate **IPv6 traffic** from ATE:Port1 to random IP addresses in
+    IPv6-DST-NET using a random combination source addresses at linerate.
+    *   Use 512 bytes frame size.
+    *   Set ToS value of *0x80* for all packets.
+    *   Set TTL of the packets to *10*.
+
+Verify:
+
+*   Policy forwarding packet counters matches the packet count of traffic
+    generated from ATE:Port1.
+*   The packet count of traffic sent from ATE:Port1 should be equal to the sum
+    of all packets egressing DUT ports 2 to 5.
+*   All packets egressing DUT ports 2 to 5 are GUE encapsulated.
+*   ECMP hashing works (equal traffic) over the two LAG interfaces.
+*   LAG hashing works (equal traffic) over the two Singleton ports.
+*   ToS for all GUE encapsulated packets:
+    *   GUE header ToS is **0x80**.
+    *   Inner header Traffic Class is **0x80**.
+*   TTL for all GUE encapsulated packets:
+    *   GUE header TTL is **9**.
+    *   Inner header Hop Limit is **9**.
+
+### PF-1.3.3: IPv4 traffic GUE encapsulation with explicit ToS configuration on tunnel
+
+DUT and ATE actions:
+
+*   Re-configure the IPv4 GUE tunnel on the DUT with ToS value *0x60*.
+*   Generate **IPv4 traffic** from ATE:Port1 to random IP addresses in
+    IPv4-DST-NET using a random combination source addresses at linerate.
+    *   Use 512 bytes frame size.
+    *   Set ToS value of *0x80* for all packets.
+    *   Set TTL of the packets to *10*.
+
+Verify:
+
+*   Policy forwarding packet counters matches the packet count of traffic
+    generated from ATE:Port1.
+*   The packet count of traffic sent from ATE:Port1 should be equal to the sum
+    of all packets egressing DUT ports 2 to 5.
+*   All packets egressing DUT ports 2 to 5 are GUE encapsulated.
+*   ECMP hashing works (equal traffic) over the two LAG interfaces.
+*   LAG hashing works (equal traffic) over the two Singleton ports.
+*   ToS for all GUE encapsulated packets:
+    *   GUE header ToS is **0x60**.
+    *   Inner header ToS is **0x80**.
+*   TTL for all GUE encapsulated packets:
+    *   GUE header TTL is **9**.
+    *   Inner header TTL is **9**.
+
+### PF-1.3.4: IPv6 traffic GUE encapsulation with explicit ToS configuration on tunnel
```

Comment:
Same clarifications as noted in `PF-1.3.3`. PTAL at adding these throughout the following test cases.

Reply:
Ack, updated all the test cases with the clarifications from other test cases.